crates/polars-ops/src/series/ops/arg_min_max.rs

```diff
@@ -195,24 +212,38 @@ fn first_unset_bit(mask: &Bitmap) -> usize {
 }
 
 fn arg_min_str(ca: &Utf8Chunked) -> Option<usize> {
+    if ca.is_empty() || ca.null_count() == ca.len() {
+        return None;
+    }
     match ca.is_sorted_flag() {
-        IsSorted::Ascending => Some(0),
-        IsSorted::Descending => Some(ca.len() - 1),
+        IsSorted::Ascending => ca.first_non_null(),
```

Comment:
Perhaps we can test `ca.null_count() == 0`, and if it doesn't hold up, we can go here. Otherwise, we will still follow the previous logic?

Reply:
I don't think it would matter much in performance. I leave this up to you.